browser-test/src/support/civiform_fixtures.ts

```diff
@@ -30,6 +31,7 @@ type CiviformFixtures = {
   adminSettings: AdminSettings
   applicantFileQuestion: ApplicantFileQuestion
   applicantProgramList: ApplicantProgramList
+  programOverview: ApplicantProgramOverview
```

Comment:
```suggestion
  applicantProgramOverview: ApplicantProgramOverview
```

(blocking) Please rename so this matches up with the same naming pattern.

Reply:
Good idea.  Done.